### PR TITLE
Use CentOS5 based docker image on CircleCI

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

File: conda_smithy/configure_feedstock.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def compute_build_matrix(meta):
 
 def main(forge_file_directory):
     recipe_dir = 'recipe'
-    config = {'docker': {'image': 'pelson/obvious-ci:latest_x64', 'command': 'bash'},
+    config = {'docker': {'image': 'jjhelmus/conda-forge-x86_64:latest', 'command': 'bash'},
```
Comment (pelson) [inline conda_smithy/configure_feedstock.py]: I'd be more comfortable if this were under a conda-forge organisation on dockerhub. Does such a concept exist? Would you mind setting it up and adding yourself, me, @msarahan and @ocefpaf as administrators?

Comment (jjhelmus) [inline conda_smithy/configure_feedstock.py]: This should be possible. I made a condaforge organization on dockerhub (it did not like the hyphen) and added everyone as owners.  Network issues are holding be back from pushing the image up the the organization but it should be there later today.

**Comment by pelson** [comment]
Same comment as https://github.com/conda-forge/staged-recipes/pull/97#issuecomment-197764986. Otherwise :+1: on this change.

In terms of rolling this out across the feedstock estate, I have a script which can propose the change as a PR to each repo. I'd rather not flood the CI systems, so I delay each PR, and ideally, we wouldn't do it too often so it would be helpful to get a feeling for how urgently we should push this out (vs waiting for other optimisations/features).

Cheers!

**Comment by janschulz** [comment]
AFAIK "[ci skip]" will skip on all ci systems... Why not use that?

**Comment by pelson** [comment]
> AFAIK "[ci skip]" will skip on all ci systems

I think that is just AppVeyor's standard atm. I saw no mention if it elsewhere.

**Comment by pelson** [comment]
Plus, we *do* actually want the CI to run so that we can catch anything that gets broken by a change (that is the main reason for having individual feedstocks).

**Comment by janschulz** [comment]
`[skip ci]` or `[ci skip]` support:

* Circle CI: https://circleci.com/docs/skip-a-build
* Appveyor: https://www.appveyor.com/docs/how-to/skip-build
* Travis: https://docs.travis-ci.com/user/customizing-the-build/#Skipping-a-build

Appveyor seems to be the only one which can be skipped specifically with `[appveyor skip]`. There is a bug for travis: https://github.com/travis-ci/travis-ci/issues/5032 and it should be possible to implement such a thing on our own: https://github.com/travis-ci/travis-ci/issues/5032#issuecomment-174352520 (the latter should probably also work on CircleCI as long as there is a env var with the commit message available)

**Comment by pelson** [comment]
Oooo. Thanks @janschulz very handy to know! :+1:

**Comment by jakirkham** [comment]
Does this still need to be open? If we were to switch to CentOS 5, it seems that change would need to occur to our Docker image instead of switching where the image comes from.

**Comment by jjhelmus** [comment]
We've settled on CentOS 6 for a build platform, this can be closed.

**Comment by jakirkham** [comment]
Thanks @jjhelmus.

**Pull request closed without merging**